include/variables.h

```diff
@@ -79,7 +79,7 @@ extern float D_80097F90;
 // extern UNK_TYPE1 D_80097FA0;
 // extern UNK_TYPE1 D_80097FA4;
 // extern UNK_TYPE1 D_80097FA5;
-// extern UNK_TYPE4 D_80097FB0;
+extern UNK_TYPE4 __osPfsLastChannel;
```

Comment:
You know the type of this variable
```suggestion
extern s32 __osPfsLastChannel;
```